src/components/keyboard.js

```diff
@@ -0,0 +1,438 @@
+import TextArea from './text-area';
+import Key from './key';
+
+export default class Keyboard {
+  constructor(selectorStr) {
+    this.text = new TextArea();
+
+    this.langList = ['en', 'ru'];
+    this.setLang();
+    this.setDefaultControlKeyState();
+
+    this.keys = [];
+    this.keyMap = {};
+    this.selector = document.querySelector(selectorStr);
+
+    this.buildKeyBoard();
+    this.addEventListeners();
+  }
+
+  setLang(langIndex) {
+    if (typeof window.localStorage.lang === 'undefined') {
```

Comment:
Do you need typeof here? Why can't you check just value?

Reply:
> Your keyboard layout looks strange because all keys have different width. You can check https://blog.wooting.nl/the-ultimate-guide-to-keyboard-layouts-and-form-factors/ Also I saw you design in figma and it doesn't match with your application.

Yes, I used Figma for sketch)
As a result, design was already being finalized in the CSS.